paasta_tools/kubernetes_tools.py

```diff
@@ -815,14 +821,23 @@ def get_kubernetes_services_running_here_for_nerve(
     nerve_list = []
     for kubernetes_service in kubernetes_services:
         try:
-            kubernetes_service_config = load_kubernetes_service_config(
-                service=kubernetes_service.name,
-                instance=kubernetes_service.instance,
-                cluster=cluster,
-                load_deployments=False,
-                soa_dir=soa_dir,
-            )
-            for registration in kubernetes_service_config.get_registrations():
+            # first try getting the list of registrations from
+            # the pods annotations. this is helpful because it doesn't
+            # require a paasta config like kuberentes-<cluster>.yaml
+            # if that fails we fall back to looking up registrations
+            # in the config file
+            if kubernetes_service.registrations:
```

Comment:
Wondering, maybe we should use the merge of them...

Reply:
Hmm, maybe I should go further and just ditch the old method...